extensions/interrupts/README.md

```diff
@@ -49,12 +49,13 @@ The following table specifies which flag is associated with which bit in the `FL
 | Illegal Instruction      | Synchronous  | 2                | Occurs when execution of a reserved, or illegal instruction is attempted.                                          |
 | Memory Alignment Error   | Synchronous  | 3                | Occurs when an attempt to read or write to memory at an unaligned address occurs and is not supported.             |
 | General Protection Fault | Synchronous  | 4                | Occurs when the CPU needs the operating system to handle an unexpected event for stability or consistency reasons. |
-| External Interrupt       | Asynchronous | 5                | Occurs when an external interrupt occurs (eg. keyboard, mouse, etc.).                                              |
+| Divide Error             | Synchronous  | 5                | Occurs when a division by zero is attempted.                                                                       |
```

Comment:
Would it be better to specify a sentinel value for divisions by 0 instead of an interrupt? Interrupts seem like a pretty heavy-handed way of dealing with them.

Reply:
RISC-V is the only ISA I can find that gives a sentinel value. The sentinel value is -1. ARM _supports_ doing this, but the default action seems to be an exception.

At the end of the day, a program that causes this to happen is erroneous and should be killed. Providing a sentinel value could hide bugs at best and cause very wrong behavior at worst. I think RISC-Vs decision is guided by the fact that they don't want the M extension to depend on interrupts.